index.html

```diff
@@ -29,196 +29,199 @@
           </ul>
         </div>
       </nav>
+      <section class="main">
+        <div class="content">
+          <h1>
+            I'm Marie Grace. <br> Glad to see you!
+          </h1>
+          <p class="presentation">
+            I'm a software developer and I can help you build a product , feature
+            or website. Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don't hesitate to
+            contact me.
+          </p>
+          <p class="connect">LET'S CONNECT</p>
+          <div class="icons">
+            <ul>
+              <li>
+                <a href=""><i class="fa fa-twitter"></i></a>
+              </li>
+              <li>
+                <a href=""><i class="fa fa-linkedin"></i></a>
+              </li>
+              <li>
+                <a href=""><i class="fa fa-medium"></i></a>
+              </li>
+              <li>
+                <a href=""><i class="fa fa-github"></i></a>
+              </li>
+              <li>
+                <a href=""><i class="fa fa-angellist"></i></a>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </section>
```

Comment:
According to the Figma design, I suggest you stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the info of the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

>### Your Design
>
><img width="1398" alt="Screenshot 2023-05-05 at 1 55 17 PM" src="https://user-images.githubusercontent.com/91237525/236533393-7627bc10-cd44-4655-9f61-fa8182ef9bdc.png">
>
>### The Figma Design
>
><img width="583" alt="Screenshot 2023-05-05 at 2 01 25 PM" src="https://user-images.githubusercontent.com/91237525/236533719-23e97e3b-f662-4a67-ac1b-c497e65a6e35.png">